### PR TITLE
fix: Move proxy address config to app.conf

### PR DESCRIPTION
**Pull request by WindSpiritSR**
Deployed at: https://door.leviatan.cn

You can signup from https://forum.leviatan.cn, then try to link github account and signin with github provider, this pr will work if these are successful

Signed-off-by: WindSpiritSR <simon343riley@gmail.com>

### CODE REVIEW

File: controllers/util.go [hunk 1]
```diff
@@ -33,9 +33,14 @@ func InitHttpClient() {
 		return
 	}
 
+	httpProxy := beego.AppConfig.String("httpProxy")
+	if len(httpProxy) == 0 {
```
Comment (hsluoyz) [inline controllers/util.go:37]: httpProxy == ""

File: conf/app.conf [hunk 2]
```diff
@@ -7,5 +7,6 @@ driverName = mysql
 dataSourceName = root:123@tcp(localhost:3306)/
 dbName = casdoor
 authState = "casdoor"
+httpProxy = "127.0.0.1:10808"
 useProxy = false
```
Comment (hsluoyz) [inline conf/app.conf:11]: Then `useProxy` can be removed.

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by WindSpiritSR** [comment]
fixed

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**